src/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixer.php

```diff
@@ -159,7 +161,7 @@ private function fixSpacesAroundParentheses(Tokens $tokens)
             }
 
             if ($token->isType(DocLexer::T_NONE)) {
-                if (false !== strpos($token->getContent(), "\n")) {
+                if (false !== strpos($token->getContent(), $lineEnding)) {
```

Comment:
This fixer is **not** adding any newline (nor indent)/
What is the reason to make it `WhitespacesAware` ?

All tests works without that line change.